pkg/plugins/runtime/k8s/webhooks/injector/testdata/inject.sidecar-feature.02.golden.yaml

```diff
@@ -29,13 +29,6 @@ spec:
       name: default-token-w7dxf
       readOnly: true
   initContainers:
-  - command:
```

Comment:
why is this golden file changing since it's not related to native sidecars? https://github.com/kumahq/kuma/blob/aa9e2a6e23970d45de3d81767b85558015dd2449/pkg/plugins/runtime/k8s/webhooks/injector/injector_test.go#L167-L182

does it matter?

Reply:
It's the "sidecar enabled" golden file. The filename is different

Reply:
This PR changes the order so that init containers come after `kuma-sidecar`.

Reply:
@slonka does that make sense?